sale_delivery_state/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com).
+# @author Pierrick BRUN <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale delivery State",
+    "summary": "Show the delivery state on the sale order",
+    "version": "10.0.1.0.0",
+    "category": "Product",
+    "website": "www.akretion.com",
```

Comment:
@yvaucher Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)